UMN-AutoPackager/public/Set-GlobalConfig.ps1

```diff
@@ -0,0 +1,85 @@
+function Set-Config {
```

Comment:
See Get-Config, this name should be changed to match the decision made on that name.

Reply:
Done.